__tests__/api.test.js

```diff
@@ -219,3 +219,39 @@ describe("/POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/PATCH /api/articles/:article_id", () => {
+  const updateVotes = {
+    inc_votes: 10,
+  };
+  it("status:200, responds with updated article", () => {
```

Comment:
Nice test. You have not checked that votes can be decreased so you need another test for this endpoint

Reply:
Done! It will be included my next pull request task 11. thank you!